ccdproc/tests/test_image_collection.py

```diff
@@ -434,6 +438,13 @@ def test_initializing_from_table_file_that_does_not_exist(self,
             ic = image_collection.ImageFileCollection(location='dasifjoaurun',
                                          info_file='iufadsdhfasdifre')
 
+    def test_no_fits_files_in_collection(self,tmpdir):
+        with catch_warnings(AstropyUserWarning) as warning_lines:
+            assert("No fits files in the collection.")
+
+
```

Comment:
One more nitpick -- could you please remove all but one of the blank lines here?